Dockerfile

```diff
@@ -14,7 +14,7 @@ FROM ubuntu:18.04
 COPY --from=builder /app/_build/prod/rel/aeternity /home/aeternity/node
 
 # OpenSSL is shared lib dependency
-RUN apt-get -qq update && apt-get -qq -y install libssl1.0.0 curl libsodium23 \
+RUN apt-get -qq update && apt-get -qq -y install libssl1.0.0 curl libsodium23 libgmp-dev \
```

Comment:
could please try a runtime package only without -dev?

Reply:
@dincho I blame @tolbrino for everything, just doing what he told me :-)

Reply:
I guess it should be `libgmp10` then? 🤔

Reply:
Fixed